integration/benchmark/contLoadGenerator.go

```diff
@@ -82,7 +83,7 @@ type LoadParams struct {
 
 // New returns a new load generator
 func New(
-	// TODO(rbtz): use context
+// TODO(rbtz): use context
```

Comment:
need to run the go format?

Reply:
go formatter will probably complain if you add the space back.  maybe move this comment above `func New(` to make the indentation less jarring